airbyte-integrations/connectors/source-monday/source_monday/spec.json

```diff
@@ -72,6 +72,14 @@
             }
           }
         ]
+      },
+      "board_ids": {
+        "type": "array",
+        "title": "Boards to sync",
+        "description": "The IDs of the boards to sync for Items and Boards streams.",
```

Comment:
Small adjustment to make things more clear:
`The IDs of the boards that the Items and Boards streams will extract records from. When left empty, streams will extract records from all boards that exist within the account.`